src/lib/storage/base_column.hpp

```diff
@@ -32,6 +32,9 @@ class BaseColumn : private Noncopyable {
   // returns the number of values
   virtual size_t size() const = 0;
 
+  // Returns the physical size of the column
+  virtual size_t data_size() const = 0;
```

Comment:
Does this take nasty things like strings into account?

Reply:
No, thanks for reminding me of strings. I have removed these methods for now.